src/components/Table/Message.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactNode } from 'react';
+
+import { Spinner } from '../LoadingIndicator';
+import { IconFolderClosed, IconAlertCircle } from '../Icons';
+
+import styles from './Message.module.scss';
+
+/*************************
+ * TableMessage
+ ************************/
+
+interface TableMessageProps {
+  children: ReactNode;
+}
+
+const TableMessage = ({ children }: TableMessageProps) => {
+  return <div className={styles.Container}>{children}</div>;
+};
+
+/*************************
+ * Table Status Message
+ ************************/
+
+export enum TableStatus {
+  Loading = 'loading',
```

Comment:
nit: uppercase if we decide to change the enum members that have been commented on.